addon-self-administration-plugin/src/main/java/org/osiam/addons/self_administration/plugin/impl/PluginImpl.java

```diff
@@ -92,37 +92,43 @@ private AccessToken getAccessToken() {
 
     private OsiamConnector getOsiamConnector() {
         if(connector == null){
-            connector = new OsiamConnector.Builder()
-                .withEndpoint(getOsiamEndpoint())
-                .setClientId(getClientId())
-                .setClientSecret(getClientSecret())
-                .build();
+            OsiamConnector.Builder builder = new OsiamConnector.Builder()
+                    .setClientId(getClientId())
+                    .setClientSecret(getClientSecret());
+
+            if (getOsiamVersion().startsWith("2")) {
```

Comment:
Just in case, so we don't forget it: https://github.com/osiam/examples/issues/28